### DISCUSSION TITLE
How to capture logs?

### DISCUSSION THREAD

**Discussion by JaredRichardWilliam** [xanderfrangos/twinkle-tray#639] [Q&A]
I want to provide them when https://github.com/xanderfrangos/twinkle-tray/discussions/638#discussion-5531301 isn't being detected, so that I can help fix the issue.

**Comment by xanderfrangos** [comment]
Assuming you are **not** using the Microsoft Store version of Twinkle Tray, you can print out a console log by running the following from Command Prompt: `"%LocalAppData%\Programs\twinkle-tray\Twinkle Tray.exe" -console`

Twinkle Tray must not already be running when executing that.
